resample/jackknife.py

```diff
@@ -1,31 +1,50 @@
-from typing import Callable
+"""
+Jackknife resampling.
+"""
 
+from typing import Callable, Sequence
 import numba as nb
 import numpy as np
 
 
-@nb.njit
-def _jackknife_generator(a: np.ndarray) -> np.ndarray:
-    n = len(a)
-    x = np.empty(n - 1, dtype=a.dtype)
-    for i in range(n - 1):
-        x[i] = a[1 + i]
-    # yield x0
-    yield x.view(x.dtype)  # must return view to avoid a numba life-time bug
+def resample(sample: Sequence) -> np.ndarray:
```

Comment:
Thinking about this some more I think it's probably okay / good to add to these to the API as I could see someone wanting to use them on their own, so I'd say it's more about making the hand-off of data to these functions less visible when they aren't being used directly.

Reply:
Actually, I do see that this was mentioned in an issue last month, must have slipped my mind